app/helpers/audit_helper.rb

```diff
@@ -60,6 +68,8 @@ def activity(type_of_activity, args = nil)
       "Sent: validation request (other validation##{args})"
     when "other_change_validation_request_received"
       "Received: request for change (other validation##{args})"
+    when "validation_requests_sent"
```

Comment:
can't we use an event per request and use the already present:

```
./audit_helper.rb:43:    when "description_change_validation_request_sent"
./audit_helper.rb:45:    when "replacement_document_validation_request_sent"
./audit_helper.rb:47:    when "additional_document_validation_request_sent"
./audit_helper.rb:49:    when "red_line_boundary_change_validation_request_sent"
./audit_helper.rb:67:    when "other_change_validation_request_sent"
```

?

Reply:
this is so we only add one audit entry for all the requests that were sent at the same time - that's why the method creates an array outside of the loop so we don't repeat the exact same audit entry, instead just ending up with:
![image](https://user-images.githubusercontent.com/9452321/135258386-698ada28-d88b-4240-bdad-b76aee98333c.png)

Reply:
the code is a bit ugly but it does achieve this purpose - I'm sure it could be refactored somehow this way we avoid creating audit entries unnecessarily when we can (each audit is a db entry as well) - plus we capture each change at the time of controller interaction, so we'd need some extra logic to grab the past audit action and pop it in as an audit argument... can of worms really

Reply:
yes OK. I thought it'd be nicer to have each one separately, it's finer-grained which makes sense for the audit, but if the AC is to group them happy with it.